kong/clustering.lua

```diff
@@ -289,6 +301,9 @@ local function communicate(premature, conf)
 
         elseif typ == "pong" then
           ngx_log(ngx_DEBUG, "received PONG frame from control plane")
+
+        else
+          ngx_log(ngx_DEBUG, "received UNKNOWN frame from control plane")
```

Comment:
The type of frame can be printed here. Maybe:
```suggestion
          ngx_log(ngx_DEBUG, "received " .. tostring(typ) .. " frame from control plane")
```